hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/blockmanagement/TestDatanodeManager.java

```diff
@@ -1009,13 +1011,25 @@ private void verifyPendingRecoveryTasks(
           .thenReturn(tasks);
     }
 
-    if (numECBlocks > 0) {
+    if (numEcBlocksToBeReplicated > 0) {
+      Mockito.when(nodeInfo.getNumberOfECBlocksToBeReplicated())
+              .thenReturn(numEcBlocksToBeReplicated);
+
+      List<BlockTargetPair> ecReplicatedTasks =
+              Collections.nCopies(
+                      Math.min(numECTasksToBeReplicated, numEcBlocksToBeReplicated),
+                      new BlockTargetPair(null, null));
+      Mockito.when(nodeInfo.getReplicationCommand(numECTasksToBeReplicated))
```

Comment:
```suggestion
      Mockito.when(nodeInfo.getErasureCodeCommand(numECTasksToBeReplicated))
```

Reply:
Ok, there should be 'Mockito.when(nodeInfo.getECReplicatedCommand(numECTasksToBeReplicated))'.